### ISSUE TITLE
Lab Report 4 Feedback

### ISSUE THREAD

**Issue by keshab-agarwal** [vatsalbagri1102/cse15l-lab-reports#5]
Here is your feedback for Lab Report 4:
 
- [x] Provides link to their markdown-parse repository
- [x] Provides link to the markdown-parse repository they reviewed
 
Snippet 1
- [x] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program
 
Snippet 2
- [x] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program
 
Snippet 3
- [x] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program
 
Additional comments from your grader:
 Please check the expected outputs for your tests. You are using the link descriptions instead of the actual links itself in the expected outputs. Please update the tests correctly. 

### Please provide the correct links for the submissions

Feel free to comment below any questions about the rubric/feedback
 
--------
 
If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:
 
Link to this issue
Link to your published lab report page

**Comment by vatsalbagri1102** [comment]
I don't fully understand what other details to add in my screenshots for the lab report. The expected output was checked using the website given. How else must I use it?

**Comment by keshab-agarwal** [comment]
> I don't fully understand what other details to add in my screenshots for the lab report. The expected output was checked using the website given. How else must I use it?

[link](something.com)

The expected output for the getLinks function for the link tag above is "something.com" and not "link". You have the link descriptions instead of the actual links as the expected output.

**Comment by vatsalbagri1102** [comment]
So if I fix the expected output and replace the screenshots, will it earn me full credit? Or are there any other changes apart from that which need to be made too?

**Comment by vatsalbagri1102** [comment]
Also, I have received 8/14 through GitHub Issues but 5/14 on Gradescope. What can I do about that?

**Comment by keshab-agarwal** [comment]
Yes you should be good once you replace the screenshots. Sorry about the discrepancy between Gradescope and GitHub, I have updated your grade on Gradescope. Let me know if you have any other questions.
